controllers/uninstall.go

```diff
@@ -82,6 +82,14 @@ var (
 				types.NamespacedName{Name: "management-ingress-sub", Namespace: m.Namespace},
 				schema.GroupVersionKind{Group: "apps.open-cluster-management.io", Kind: "Subscription", Version: "v1"},
 			),
+			newUnstructured(
```

Comment:
Yesterday you commented
> I'll check to see if the channel doesn't need it if we can just not apply it and then see if it isn't present.

Were you able to test that? I kind of imagined these resources would not need to be explicitly uninstalled, for the same reason that none of the others do.

Reply:
@ray-harris Any update?

Reply:
the resources are removed when taken out of the templates. removing these lines.